view/next-project/src/components/common/RecordEditModal/RecordEditModal.tsx

```diff
@@ -144,6 +144,10 @@ const RecordEditModal: FC<ModalProps> = (props) => {
     }
   }, [query, router]);
 
+  const MAX_TITLE_LENGTH = 36;
+
+  const [titleLength, setTitleLength] = useState(0);
+
   const handler =
```

Comment:
```suggestion

  const MAX_TITLE_LENGTH = 36;

  const [titleLength, setTitleLength] = useState(0);

  // フォームデータ取得後にタイトルの文字数を設定
useEffect(() => {
  setTitleLength(formData.title.length); 
}, [formData.title]);
```

タイトルの文字数の長さの初期値を取得するプログラムを追加すると、最初に表示される文字数の問題は解決できます！
これで実行してみて下さい！